tests/test_routes.py

```diff
@@ -10,30 +10,45 @@
 from unittest import TestCase
 from unittest.mock import MagicMock, patch
 from service import app
-from service.models import db
-from service.common import status  # HTTP Status Codes
+from service.models import Condition, Inventory, db
+from service.common import status
+from tests.factories import InventoryFactory  # HTTP Status Codes
 
 
+DATABASE_URI = os.getenv(
+    "DATABASE_URI", "postgresql://postgres:postgres@postgres:5432/postgres"
```

Comment:
I think we should use a unified database URI, and add the set up procedure in the README.